### DISCUSSION TITLE
Sidebery Extension Disappearing

### DISCUSSION THREAD

**Discussion by JugtheMug** [mbnuqw/sidebery#812] [Q&A]
I'm currently having an issue when I'm closing and re-opening Firefox. The Sidebery extension completely disappears and I have to re-add it to Firefox each time.

I'm not sure if I'm just missing some simple setting or if this is intentional design.

Please help... 🤷‍♂️

**Comment by mbnuqw** [comment]
This is not an intentional design and it's not controlled by Sidebery.

Does this only happen with Sidebery? If it is, how do you install Sidebery?
